examples/00_quick_start/sasrec_amazon.ipynb

```diff
@@ -37,8 +37,10 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        df2 = df.copy()
</pre><p>is this something that can be done without copying the entire dataframe? I'm thinking that if the data is large, then this can eat the memory</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Yes, I have removed <code>df2</code> and continuing with <code>df</code> itself. </p>

Reply:
<p>Also, could you remove the comment.</p>